### PR TITLE
Deprecations

### PR DESCRIPTION
**Pull request by alex-schneider**
Describe your changes!

---
<details>
    <summary>Reviewer checklist</summary>
    <ul>
        <li>Read PR description: a summary about the changes is required</li>
        <li>Changelog updated</li>
        <li>Documentation: docs/{Reference, Cli, ...}, Docker and cli help/usage</li>
        <li>Pulled branch, manually tested</li>
        <li>Verified requirements are met</li>
        <li>Reviewed the code</li>
        <li>Reviewed the related tests</li>
    </ul>
</details>

### CODE REVIEW

**Comment by malud** [comment]
Please revert the commit https://github.com/avenga/couper/pull/734/commits/5b924b7e7d762072d6b6cad68ce7725b0641bb7c since its changes almost 20 func calls to use an empty argument to get rid of 3 lines of code.

File: config/configload/deprecated_test.go [hunk 1]
```diff
@@ -0,0 +1,82 @@
+package configload
+
+import (
+	"context"
+	"testing"
+
+	"github.com/avenga/couper/config/parser"
+	"github.com/avenga/couper/internal/test"
+	"github.com/google/go-cmp/cmp"
+	"github.com/hashicorp/hcl/v2/hclsyntax"
```
Comment (johakoch) [inline config/configload/deprecated_test.go:10]:
```suggestion
	"github.com/google/go-cmp/cmp"
	"github.com/hashicorp/hcl/v2/hclsyntax"

	"github.com/avenga/couper/config/parser"
	"github.com/avenga/couper/internal/test"
```

File: config/configload/deprecated.go [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+		renamed = append(renamed, name)
+	}
+
+	return renamed
+}
+
+// In some test cases the logger is <nil>, but not in production code.
+func (d deprecated) log(name, old string, logger *logrus.Entry) {
+	if logger != nil {
+		logger.Warn(fmt.Sprintf(
```
Comment (malud) [inline config/configload/deprecated.go:136]: `Warnf` should do the trick here. Also lower case the log message.

Comment (malud) [reply config/configload/deprecated.go:136]: adapt related tests

**Review by malud — CHANGES_REQUESTED** [review]

**Review by malud — APPROVED** [review]

**Pull request merged**